build/configs/rtl8721csm/board_metadata.txt

```diff
@@ -0,0 +1,11 @@
+BOARD="RTL8721CSM"
```

Comment:
Please add license for every `board_metadata.txt`.

Reply:
Done.